research-on-biomedical-engineering.csl

```diff
@@ -63,13 +63,9 @@
     </choose>
   </macro>
   <macro name="publisher">
-    <group delimiter="; ">
-      <group delimiter=": ">
-        <text variable="publisher-place"/>
-        <text variable="publisher"/>
-      </group>
-      <text macro="year-date"/>
-    </group>
+    <text variable="publisher-place" suffix=": " text-case="title"/>
+    <text variable="publisher" suffix="; "/>
+    <text macro="year-date"/>
```

Comment:
@damnation333, you reverted all those beautiful group delimiters that @adam3smith added in the first PR. 😭